npm-shrinkwrap.json

```diff
@@ -2165,9 +2165,9 @@
       "integrity": "sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw=="
```

Comment:
Let's revert these changes since we're not changing deps.

Reply:
Nice! Good call.